app/components/appMainModule/measurePageModule/state.js

```diff
@@ -91,15 +73,8 @@ module.exports = function($stateProvider) {
         parent: 'appMain.measure.index'
       },
       resolve: {
-        opposers: function($stateParams, $q) {
-          return $q.resolve([
-            {name: 'Citizens for a Better Oakland', contributions: 185859},
-            {name: 'Oaklanders for Ethical Government', contributions: 152330},
-            {name: 'Americans for Liberty', contributions: 83199},
-            {name: 'Golden State Citizens for Positive Reform', contributions: 23988},
-            {name: 'The Public Commission for Ethical Civic Reform', contributions: 15040},
-            {name: 'The Committee of True Americans who Dearly Love America and Liberty', contributions: 7943}
-          ]);
+        opposers: function($stateParams, disclosureApi) {
+          return disclosureApi.opposing.list();
```

Comment:
Needs param (ballot_id) and eventually we want a limit, size and offset. Does djang-rest-framework support that out of the box?